src/main/java/com/synopsys/integration/jenkins/scan/service/ScannerArgumentService.java

```diff
@@ -17,23 +18,25 @@
 public class ScannerArgumentService {
     private static final String DATA_KEY = "data";
 
-    public List<String> getCommandLineArgs(Map<String, Object> scanParameters) {
+    public List<String> getCommandLineArgs(Map<String, Object> scanParameters, BitBucket bitBucket) {
         String stageName = getStageType(scanParameters);
```

Comment:
as far as i know this `getStageType` method has been removed. check with dev branch

Reply:
Synced with the dev branch. And `getStageType` method is removed.